.gitignore

```diff
@@ -1 +1,21 @@
-node_modules/
\ No newline at end of file
+node_modules/
+/server/features.csv
+/server/photos.csv
+/server/product.csv
+/server/related.csv
+/server/styles.csv
+/server/skus.csv
+# related
+server/related
+
+# features
+server/features
```

Comment:
What's stored in these folders? Is there anything necessary for deployment in there?

Reply:
yes - I have my CSV files which i should actually maybe put up in github? not sure at the moment